### PR TITLE
Refactor Sass in components

### PR DESCRIPTION
**Pull request by maya**
This refactors Sass for files in `assets/_scss/components` and follows 18F Front-end coding styleguide: https://pages.18f.gov/frontend/.

This includes removing nested properties, alphabetizing, correct nesting order, etc...

### CODE REVIEW

File: assets/_scss/components/_alerts.scss [hunk 1]
```diff
@@ -9,11 +6,13 @@
   margin-top: 1.5em;
   padding: 1em;
 
+  @include media($medium-screen) {
+    background-size: 5.2rem;
```
Comment (msecret) [inline assets/_scss/components/_alerts.scss]: Why is an `rem` unit being used here and an `em` down there? Should the `ul` margin top be changed based on a parent font size?

File: assets/_scss/components/_disclaimer.scss [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+
+  @include media($small-screen) {
+    font-size: $h5-font-size;
   }
 }
 
 // This hides the stage note (i.e. alpha, beta, etc) of the disclaimer for 
 // anything smaller than large screen sizes
 .usa-disclaimer-stage {
+  display: none;
```
Comment (msecret) [inline assets/_scss/components/_disclaimer.scss]: Does it need to appear to screen readers regardless of size? I'm actually not sure what a mobile screen reader would publish its size as.

Comment (maya) [inline assets/_scss/components/_disclaimer.scss]: A screen reader on a desktop would read it and on a mobile device would not. I don't know what the priority of the stage text is for screen reader users.

File: assets/_scss/components/_alerts.scss (hunk 1 above)
Comment (maya) [inline assets/_scss/components/_alerts.scss]: This `rem` unit is for the icon image, where as the rest applies for text. I don't think there's a need to have a huge icon image if there's larger text. Correct, in this case the margin top would change depending on the parent text size.

**Pull request merged**